src/Common/CodeCracker.Common/DiagnosticId.cs

```diff
@@ -57,6 +57,7 @@ public enum DiagnosticId
         UseConfigureAwaitFalse = 70,
         IntroduceFieldFromConstructor = 71,
         RemoveAsyncFromMethod = 72,
-        AddBracesToSwitchSections = 73
+        AddBracesToSwitchSections = 73,
+        TernaryOperator_Iif = 74
```

Comment:
`74` is taken with another analyzer (see the [diagnostic ids](https://github.com/code-cracker/code-cracker/wiki/DiagnosticIds) page on the wiki). You may grab the next number and update the @jwooley!